system-test/tpc-tests.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {after, describe, it} from 'mocha';
+import {Bigtable} from '../src';
+
+describe.skip('TPC tests', () => {
+  // These tests are only designed to pass when using the service account
+  // credentials for the TPC environment so we skip them in the CI pipeline.
+  //
+  // To see successful tests, uncomment the following line:
+  // process.env.GOOGLE_APPLICATION_CREDENTIALS = '/path-to/tpc_sa_key.json';
+
+  function runTest(done: Mocha.Done, bigtable: Bigtable) {
```

Comment:
I'd make runTest an async function here and remove the inner lambda with async. Down below, you can just make your test functions passed to mocha async and then await on runTest.

Reply:
Done